Firmware/ultralcd.cpp

```diff
@@ -7870,17 +7853,15 @@ static bool lcd_selfcheck_axis_sg(unsigned char axis) {
 	tmc2130_home_exit();
 	enable_endstops(true);
 
-	if (axis == X_AXIS) { //there is collision between cables and PSU cover in X axis if Z coordinate is too low
-		raise_z_above(17,true);
-		tmc2130_home_enter(Z_AXIS_MASK);
-		st_synchronize();
-		tmc2130_home_exit();
-	}
+
+	raise_z_above(MESH_HOME_Z_SEARCH);
+	st_synchronize();
+	tmc2130_home_enter(1 << axis);
 
 // first axis length measurement begin	
 	
 	current_position[axis] -= (axis_length + margin);
-	plan_buffer_line_curposXYZE(manual_feedrate[0] / 60, active_extruder);
+	plan_buffer_line_curposXYZE(homing_feedrate[axis] / 60, active_extruder);
```

Comment:
@vintagepc  homing_feedrate vs. manual_feedrate = 3000 vs. 2700 - since this code is being used in normal self test, which has been around for quite a while (and since the selftest is probably failing more with the 3000 feedrate), please revert the homing_feedrate back to manual_feedrate in this whole PR. Otherwise it looks reasonably well

Reply:
Done!